yarn-project/cli/aztec-cli

```diff
@@ -99,7 +99,7 @@ AZTEC_CLI_EXTRA_ARGS=""
 
 # first process commands with positional arguments
 # assumes positional arguments are the first thing
-if [[ "$AZTEC_CLI_COMMAND" == "compile" || "$AZTEC_CLI_COMMAND" == "deploy" || "$AZTEC_CLI_COMMAND" == "inspect-contract" ]]; then
+if [[ "$AZTEC_CLI_COMMAND" == "compile" || "$AZTEC_CLI_COMMAND" == "deploy" || "$AZTEC_CLI_COMMAND" == "inspect-contract" || "$AZTEC_CLI_COMMAND" == "update-aztecnr" ]]; then
```

Comment:
Definitely need a better solution for this.

Reply:
Fixed in #3138

Reply:
in cli/index.ts the command is called "update". Why is it "update-aztecnr" here?

Reply:
Fixed :)